packages/waku/src/lib/plugins/vite-plugin-rsc-index.ts

```diff
@@ -75,9 +75,6 @@ ${opts.htmlHead}
     },
     transformIndexHtml() {
       return [
-        // HACK without <base>, some relative assets don't work.
```

Comment:
Do you know what was the original problem?

Reply:
No honestly, but what I guess was the potential solution was the fact the we made things either absolute to the project root or to the user's root.

Reply:
Let me check:
- It's introduced in #289.
- https://github.com/dai-shi/waku/pull/289/commits/cb17af57774fc033d6eb8a26d410a62bad11d446
- Going back to 7c0a6ef4fc8ff3aa2d56d0728882655534d14b0d and remove the hack
- npm run website:prd
- open http://localhost:8080/blog/introducing-waku

<img width="631" alt="image" src="https://github.com/dai-shi/waku/assets/490574/42566d1f-1b18-4a57-bbff-1402f58cb1b7">

Now, let's try this branch:
- git switch fix/remove-base 
- npm run website:prd
- open http://localhost:8080/blog/introducing-waku

same error

<img width="611" alt="image" src="https://github.com/dai-shi/waku/assets/490574/2a388b57-fffd-4bef-beff-27657ca2c75b">

So, no, it's not fixed.

Reply:
will check this soon!

Reply:
@PerfectPan For sure! feel free to, i'd be happy to close this if that solves it.